vidarrtest-regression.json.in

```diff
@@ -280,7 +280,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/umiQC/output_metrics/1.1.1/umiQC_dataset1.metrics",
+                "output_metrics": "/.mounts/labs/gsi/testdata/umiQC/output_metrics/workflow_test_01.metrics",
```

Comment:
This is probably a good chance to drop **camelCase** for the workflow name (the build file says that the name of this workflow is umiqc, not umiQC) but it looks like someone decided to use the same syntax as the wdl file is using, which is not consistent.